fiftyone/utils/utils3d.py

```
@@ -28,9 +32,13 @@
 import fiftyone.core.validation as fov
 import fiftyone.utils.data as foud
 import fiftyone.utils.image as foui
+import fiftyone.core.validation as fov
```

Ruff (0.12.2)

Redefinition of unused `fov` from line 32

Remove definition: `fov`

(F811)